.idea/vcs.xml

```diff
@@ -14,7 +14,7 @@
             </list>
         </option>
     </component>
-  <component name="VcsDirectoryMappings">
-    <mapping directory="$PROJECT_DIR$" vcs="Git" />
-  </component>
+    <component name="VcsDirectoryMappings">
+        <mapping directory="$PROJECT_DIR$" vcs="Git" />
+    </component>
```

Comment:
We should not commit this

Reply:
@owaiskazi19 this is already part of `main` and `2.x` branches, should we completely remove that or keep it as it is?, this change is made by local IDE. I havent intentionally made any change.

Reply:
Yes, the file is there. I meant the formatting change. Let's remove this change from the commit.

Reply:
Done, can you please check the PR again?